bandwidth-file-spec.txt

```diff
@@ -438,6 +578,8 @@
       MaxAdvertisedBandwidth for a relay, Torflow reduces that relay's
       measured bandwidth.
 
+      This KeyValue pair MUST be present in each relay Line.
```

Comment:
Isn't this implicit from `[Exactly once.]`?

Reply:
Ok, I will revert this change.

Reply:
Fixed in 2aa120a.